site/messages/en.json

```diff
@@ -103,17 +103,17 @@
   "update-funding-rate": "Update Funding Rate",
   "unpause-stream": "Unpause Stream",
   "unwrap-weth-success": "You unwrapped {value} WETH successfully!",
-  "unwrap": "unwrap",
+  "unwrap": "Unwrap ETH",
```

Comment:
Given this is Hemi's fork I wonder if the utility `wrap ETH` or `unwrap ETH` should be `wrap hETH` and `unwrap hETH` as that's the real eth name on Hemi network [as defined here](https://github.com/hemilabs/infrastructure/blob/main/NETWORK_INFO.md#token-naming-standards-1) (for testnet, it would be `thETH`). While technically more accurate, I don't know if any user would min if the page talks about ETH.. Thoughts @gabmontes  ?

Reply:
changed here: https://github.com/hemilabs/pure.finance/pull/2/commits/9aefa58924e45d52cc6e24097b0b9dde3e03e27a

I had to add a `chain-list` package to be able to get the nativeTokenSymbol dynamically.